api/client/webclient/webconfig.go

```diff
@@ -55,6 +55,12 @@ type WebConfig struct {
 	TunnelPublicAddress string `json:"tunnelPublicAddress,omitempty"`
 	// RecoveryCodesEnabled is a flag that determines if recovery codes are enabled in the cluster.
 	RecoveryCodesEnabled bool `json:"recoveryCodesEnabled,omitempty"`
+
+	UiConfig UiConfig `json:"uiConfig,omitempty"`
+}
+
+type UiConfig struct {
```

Comment:
```suggestion
type UIConfig struct {
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms